package.json

```diff
@@ -240,7 +240,7 @@
     "kafka-avro": "^3.2.0",
     "node-rdkafka": "^3.0.1",
     "pg-native": "^3.1.0",
-    "prisma": "^5.15.1"
+    "prisma": "^5.16.0"
```

Comment:
The latest version build failed with the following error:

> PrismaClientInitializationError: error: Environment variable not found: DATABASE_URL

An issue regarding this has been reported in the repository. For more details, please see: [https://github.com/prisma/prisma/issues/24658](https://github.com/prisma/prisma/issues/24658).

Reply:
They fixed the issue with env variable in  v5.16.1.